.github/workflows/test-env-deploy.yml

```diff
@@ -129,10 +130,10 @@ jobs:
           fallback_uri: ${{ secrets.CYPRESS_API_URI }}
         run: |
           echo "::set-output name=custom_api_uri::$(echo $pull_request_body | grep -Eo "$prefix$pattern" | sed s/$prefix// | head -n 1 | { read custom_uri; if [ -z "$custom_uri" ]; then echo "$fallback_uri"; else echo "$custom_uri"; fi })"
-      - name: Get base_URL
-        id: base_URL
-        run: |
-          echo "::set-output name=base_URL::https://$(echo ${GITHUB_HEAD_REF}).dashboard.saleor.rocks"
+      #- name: Get base_URL
+      #  id: base_URL
+      #  run: |
+      #    echo "::set-output name=base_URL::https://$(echo ${GITHUB_HEAD_REF}).dashboard.saleor.rocks"
```

Comment:
This won't work if this step is commented / removed:
`CYPRESS_baseUrl: ${{ steps.base_URL.outputs.base_URL }}`

Please add `base_URL` to deploy job outputs and use  `${{needs.deploy.outputs.base_URL}}` in cypress config.

Reply:
fixed under .github/workflows/test-env-deploy.yml